lib/plugins/aws/invokeLocal/resolvers/secretsManagerSecret.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+const ServerlessError = require('../../../../classes/Error').ServerlessError;
+
+module.exports = {
+  'Fn::GetAtt': (plugin, resource, resourceName) => {
+    throw new ServerlessError(`Cannot get the attribute of ${resourceName}`);
```

Comment:
If there's no support for such resolution, I wouldn't configure this file at all

Reply:
I've removed it, but this will result in a less friendly error to the user